packages/nc-gui/components/cell/MultiSelect.vue

```diff
@@ -153,12 +159,16 @@ watch(isOpen, (n, _o) => {
         v-if="options.find((el) => el.title === val)"
         class="rounded-tag"
         :style="{ display: 'flex', alignItems: 'center' }"
-        :color="options.find((el) => el.title === val).color"
+        :color="(options.find((el) => el.title === val) as SelectOptionType).color"
```

Comment:
Please don't cast to a type in the template, in VSCode that's okay but in Webstorm this breaks the whole file for me 😅
If you need to force a type, use a function that returns the correct type for you, that way I can have less of a headache in Webstorm xD

Reply:
oh okay let me see that